src/PlusWidgets/CMakeLists.txt

```diff
@@ -40,7 +40,9 @@ SET(${PROJECT_NAME}_INCLUDE_DIRS
   ${CMAKE_CURRENT_BINARY_DIR}
   ${CMAKE_CURRENT_SOURCE_DIR}
   )
-IF(CMAKE_VERSION VERSION_GREATER_EQUAL "3.8")
+# If CMake 3.7 is the minimum version then the next line can be

```

Comment:
Yes, I would like that. 3.7.2 seems to be very stable and all the features we need.